crates/topos-tce-api/src/runtime/mod.rs

```diff
@@ -58,6 +59,7 @@ pub struct Runtime {
     pub(crate) broadcast_stream: broadcast::Receiver<CertificateDeliveredWithPositions>,
 
     pub(crate) storage: StorageClient,
+    pub(crate) transient_stream: HashMap<Uuid, Sender<Certificate>>,
```

Comment:
Could this be a tuple perhaps?

Reply:
A tuple ? for what exactly?

Reply:
I was thinking: the `transient_stream` seems to be a single stream, not a collection, so why store the value in a `HashMap` as opposed to just having a `(Uuid, Sender<Certificate>)`. But maybe it should simply be renamed to `transient_streams` (plural)?

Reply:
I was thinking: the transient_stream seems to be a single stream, not a collection, so why store the value in a HashMap as opposed to just having a (Uuid, Sender<Certificate>). But maybe it should simply be renamed to transient_streams (plural)?

Reply:
yes let's rename it to plural form